sdk/eventhub/Azure.Messaging.EventHubs/README.md

```diff
@@ -22,7 +22,7 @@ To use Azure services, including Azure Event Hubs, you'll need a Microsoft Azure
 
 To interact with Azure Event Hubs, you'll also need to have an Event Hub available.  If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [creating an Event Hub using the Azure portal](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create).  There, you can also find detailed instructions for using the Azure CLI, Azure PowerShell, or Azure Resource Manager (ARM) templates to create an Event Hub.
 
-To quickly create the needed Event Hubs resources in Azure and view your connection string, you can deploy our sample template by clicking:
+To quickly create the needed Event Hubs resources in Azure and to view your connection string, you can deploy our sample template by clicking:
 
 [![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-net%2Fmaster%2Fsdk%2Feventhub%Azure.Messaging.EventHubs%2Fassets%2Fsamples-azure-deploy.json)
 
```

Comment:
Is connection string is considered a credential? If yes, it should not be a part of the code but set as an environment variable (@ all snippets below)

Reply:
The connection string is generated by the ARM template when creating resources and returned as part of the ARM function;  no credential appears in code.